RUFAS/classes.py

```diff
@@ -86,11 +87,11 @@ def __init__(self, data, weather_file):
             weather_file: path to the weather file specified in the json file
         """
 
-        info_map = {"class": self.__class__.__name__, 
+        info_map = {"class": self.__class__.__name__,
                     "function": self.__init__.__name__,
                     "data": data,
-                    "weather_file": weather_file,}
-        
+                    "weather_file": weather_file, }
```

Comment:
It seems like `weather_file` is not actually a file, but a path string. Can we change the mapping key (and the name of the value object, if possible)?
```suggestion
                    "weather_path": weather_file, }
```

Reply:
How about "weather_file_path"?

Reply:
even better!

Reply:
I will make that change. Can you say a little more about changing the variable name for the value as well? Were you thinking for the name in the argument or just for this one instance to be stored under that name in the info_map?

Reply:
Sure, I think that it is OK to rename only the dictionary key. Ideally the entire `weather_file` object would be renamed to `weather_file_path` (unless it actually is a connection to the file and not a string), but that would require changing it throughout the entire repo. I'm not sure if such a minor gripe is worth your time (or anyone else's) right now.